### PR TITLE
Update readme with People FAQ 

### PR DESCRIPTION
**Pull request by alabiaga**

### CODE REVIEW

**Comment by alabiaga** [comment]
cc\ @yaraki Yuichi pls review. I removed some sections that are still in review and will just add them when finalized. Thanks

File: People/README.md [hunk 1]
```diff
@@ -68,6 +68,94 @@ on Android 11 or later.
 This sample uses the Gradle build system. To build this project, use the
 "gradlew build" command or use "Import Project" in Android Studio.
 
+## FAQ
+
+### Bubbles
+
+**Q. How can one determine within an activity, that it is in an expanded bubble?**
+
+    class BubbleActivity : AppCompatActivity() {
```
Comment (yaraki) [inline People/README.md:77]: Surround this code block with ```.
https://docs.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks

File: People/README.md [hunk 2]
```diff
@@ -68,6 +68,94 @@ on Android 11 or later.
...
+      override fun onCreate(...) {
+      ...
+      val displayId = windowManager.defaultDisplay.displayId
+      val mIsBubbled = displayId != Display.DEFAULT_DISPLAY
```
Comment (yaraki) [inline People/README.md:81]: mIsBubbled → isBubbled
(Also in the inline below)

File: People/README.md (hunk 1 above)
Comment (alabiaga) [reply People/README.md:78]: done

File: People/README.md (hunk 2 above)
Comment (alabiaga) [reply People/README.md]: done

File: People/README.md (hunk 1 above)
Comment (alabiaga) [reply People/README.md:78]: done

**Pull request merged**